### PR TITLE
Chapter5 user signup

### PR DESCRIPTION
**Pull request by leducson**
selection_007

### CODE REVIEW

**Comment by leducson** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,9 +6,13 @@
     <%= stylesheet_link_tag "application", media: "all",
                                               "data-turbolinks-track": "reload" %>
     <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+    <%= render "shared/shim"%>
   </head>
-
   <body>
-    <%= yield %>
+    <%= render "shared/header"%>
```
Comment (hhhhuy112) [inline app/views/layouts/application.html.erb]: space

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -6,9 +6,13 @@
...
+    <div class="container">
+      <%= yield %>
+      <%= render "shared/footer"%>
```
Comment (hhhhuy112) [inline app/views/layouts/application.html.erb]: space

**Comment by hhhhuy112** [comment]
commented

**Comment by leducson** [comment]
ready

**Comment by phamvanbk11** [comment]
merged
Bữa sau tạo 1 pull 1 commit thôi nhé em

**Pull request merged**